test/Utilities/Microsoft.Testing.TestInfrastructure/GlobalUsings.cs

```diff
@@ -0,0 +1,5 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma warning disable IDE0005 // Using directive is unnecessary.
+global using Polyfills;
```

Comment:
Can you add the import in the csproj instead please.

Reply:
done